charts/jaeger/templates/es-index-cleaner-cronjob.yaml

```diff
@@ -53,6 +53,16 @@ spec:
             args:
               - {{ .Values.esIndexCleaner.numberOfDays | quote }}
               - {{ include "elasticsearch.client.url" . }}
+              {{- range $key, $value := .Values.esIndexCleaner.cmdlineParams -}}
```

Comment:
instead of this being repeated, why don't we create a helper template variable similar to [elasticsearch.cmdargs](https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger/templates/_helpers.tpl#L413)

Reply:
true, I thought about that after the PR was done, I change/test that next week when I have some time

Reply:
Thanks